src/wymeditor/editor/image-handler.js

```diff
@@ -367,21 +389,13 @@ WYMeditor.ImageHandler.prototype._placeResizeHandleOnImg = function (img) {
         'margin-left': '-' + IMAGE_PADDING
     });
 
-    // the resize handle, prepended to the body in this way,
-    // can be removed from the body using DOM manipulation
-    // such as setting the content with the `html` method.
-    // so we place it there in case that occurred.
-    // this could be done conditionally
-    // but there is practically no performance hit so keeping it simple
-    ih._$resizeHandle.prependTo(ih._wym.$body());
-
     // it is important that the resize handle's offset
     // is updated after the above style modification
     // adds top padding to the image
     // because that alters the image's outside height
     ih._correctResizeHandleOffsetAndWidth();
 
-    ih._$resizeHandle.show();
+    jQuery(wym.element).trigger(WYMeditor.EVENTS.postBlockMaybeCreated, wym);
```

Comment:
The triggering of this event throughout this module is kind of hackish, because the name of the event is obviously not correct.

Other than that, it seems to do what we desire, which is that the seamless skin will recalculate its height.